com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyTextDocumentService.java

```diff
@@ -262,4 +294,16 @@ private String createDescriptiveErrorMessage(String action, String uri) {
   private BiConsumer<Object, Throwable> reportExceptionIfThrown(String message) {
     return (res, ex) -> ofNullable(ex).ifPresent(it -> log.error(message, it));
   }
+
+  private Path getDependencyFileFolderPath() {
+    return Paths.get(
+        getWorkspaceFoldersAsPathList(getWorkspaceFolders()).get(0)
+            + filesystemSeparator()
+            + COBDEPS
+            + filesystemSeparator());
+  }
```

Comment:
I think we can extend the ```CopybookDependencyService``` APIs and move this implementation there, if I didn't miss anything 👼

Reply:
it can be done